app/views/dashboards/index.html.erb

```diff
@@ -23,18 +23,43 @@
 </div>
 
   <h3>Latest Preview Version
-    <% if Version.buildable? %>
-      <%= link_to 'Generate New Preview Version', dashboard_build_path, method: :post,
-          data: { confirm: 'Are you sure you want to create a new preview version?' },
-          class: "btn btn-warning btn-xs",
-          role: "button",
-          id: "version_build"
+    <div id="preview_dialog" title="Generate New Preview Version">
+      <div id="preview_dialog_div">
+        <p class="generate_preview_question">
+          <b>Are you sure you want to create a new preview version?</b>
+        </p>
+        <% if @latest_uploads.length > 0 %>
```

Comment:
`@latest_uploads` will be `nil` if `Version.current_production` is `nil` (see `Upload.latest_uploads`)

Reply:
Method returns empty array now if production is blank